doc/DeveloperJourney.md

```diff
@@ -33,15 +33,15 @@ Welcome to our tutorial! If you're interested in building your own language runt
 
 ### 1.1 A Brief Overview of Base 9
 
-If you want to fiddle around with Base9, check out our [set-up page](https://github.com/arianneb/Base9/blob/documentation/doc/README.md) for set-up instructions.
+If you want to fiddle around with Base9, check out our [set-up page](https://github.com/b9org/b9/blob/master/doc/SetupBase9.md) for set-up instructions.
 
 Base9 is an educational **language runtime**. It's front-end language, b9porcelain, is a small subset of JavaScript. It compiles into a simple set of **bytecodes** which run on a primitive **interpreter**. We've also plugged in [OMR](https://www.eclipse.org/omr/) and [JitBuilder](https://developer.ibm.com/open/2016/07/19/jitbuilder-library-and-eclipse-omr-just-in-time-compilers-made-easy/) to provide our runtime with a **JIT compiler**! 
 
 Base9 has several major components that we'll discuss throughout the course of this tutorial. We'll try to provide insight about why we made the design decisions that we did, and how we built up the pieces. We encourage you to remember that much of our implementation is made up of design decisions suited to our project. Along the way, you may wish to deviate from these decisions in order to best suit your own project.
 
 ### Base9 Overview:
 
-![](https://github.com/arianneb/Base9/blob/developerQuest/images/b9overview.png)
+![](https://github.com/b9org/b9/blob/master/images/b9overview.png)
```

Comment:
You should add an alt-text to picture links, and again definitely use relative paths.